manager/ui/war/plugins/api-manager/html/admin/admin-gateways.html

```diff
@@ -49,7 +51,7 @@ <h2 class="title" data-field="heading" apiman-i18n-key="gateways">Gateways</h2>
                     <tr ng-hide="gateways.length > 0">
                       <td colspan="2">
                         <div class="apiman-no-content">
-                          <p class="apiman-no-entities-description" apiman-i18n-key="no-gateways-found">No gateways have been added! Try adding a Gateway by clicking 'New Gateway' above - you'll need at least one, otherwise users will not be able to publish their APIs.</p>
+                          <p class="apiman-no-entities-description" apiman-i18n-key="no-gateways-found">No gateways have been added! Try adding a gateway by clicking <b>New Gateway</b> above - you'll need at least one, otherwise users will not be able to publish their APIs.</p>

```

Comment:
Please use `<em>` or `<strong>` rather than `<b>`. You can also look at bootstrap bold if this is not sufficient, but one of the aforementioned should be OK.

`<b>` is no longer recommended.

I know there are a few `<b>` sprinkled around, but we should not add any more 😅.